maven-jahia-plugin/src/main/java/org/jahia/utils/maven/plugin/osgi/models/JahiaDepends.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * ==========================================================================================
+ * =                            JAHIA'S ENTERPRISE DISTRIBUTION                             =
+ * ==========================================================================================
+ *
+ *                                  http://www.jahia.com
+ *
+ * JAHIA'S ENTERPRISE DISTRIBUTIONS LICENSING - IMPORTANT INFORMATION
+ * ==========================================================================================
+ *
+ *     Copyright (C) 2002-2021 Jahia Solutions Group. All rights reserved.
+ *
+ *     This file is part of a Jahia's Enterprise Distribution.
+ *
+ *     Jahia's Enterprise Distributions must be used in accordance with the terms
+ *     contained in the Jahia Solutions Group Terms &amp; Conditions as well as
+ *     the Jahia Sustainable Enterprise License (JSEL).
+ *
+ *     For questions regarding licensing, support, production usage...
+ *     please contact our team at sales@jahia.com or go to http://www.jahia.com/license.
+ *
+ * ==========================================================================================
+ */
+package org.jahia.utils.maven.plugin.osgi.models;
+
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * Parser for jahia-depends value
+ */
+public class JahiaDepends {
+
+    private String moduleName = "";
+    private String minVersion = null;
+    private String maxVersion = null;
+
+
+    public JahiaDepends(String dependency) {
+        String[] deps = dependency.split("=");
+        this.moduleName = StringUtils.isNotBlank(deps[0]) ? deps[0].trim() : "";
+
+        // parse version
+        if (deps.length > 1 && StringUtils.isNotEmpty(deps[1])) {
+            String[] version = deps[1].split(",");
```

Comment:
You need to support included (`[` `]`) /excluded (`(` `)`) ranges, and the "atleast" format (no `[` `]` `(` `)` or `,`). https://docs.osgi.org/specification/osgi.core/7.0.0/framework.module.html#i3189032 . It will be much easier and safer to copy a standard implementation. Preferably org.osgi.framework.VersionRange - , or maven one, but i think maven's behaviour is slightly different from OSGi (especially on the atleast, not sure it's actually the same) . OSGi one also has a `toFilterString` method. Both are in apache license and be copied as is, if you don't have access to them.

Reply:
Modified parsing to be done by `aQute.bnd.version.VersionRange` (There was an issue importing `org.osgi.framework.VersionRange` when running tests so using the bndtools version as a workaround. But I believe the bndtools version uses the osgi library internally so should be the same behaviour).

Inclusive and exclusive ranges now work, Lower-bound only versions should be declared of the form `module-name=<lower-bound version number>`